packages/create-svelte/templates/skeletonlib/package.template.json

```
@@ -3,8 +3,19 @@
 	"version": "0.0.1",
 	"scripts": {
 		"dev": "vite dev",
-		"build": "svelte-kit sync && svelte-package",
-		"prepublishOnly": "echo 'Did you mean to publish `./package/`, instead of `./`?' && exit 1"
+		"build": "vite build && npm run package",
+		"package": "svelte-kit sync && svelte-package"
+	},
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"svelte": "./dist/index.js",
+			"default": "./dist/index.js"
+		},
+		"./package.json": "./package.json"
```

I think we can lose this now that rollup-plugin-svelte is updated?

```suggestion
		}
```

---

Yes and now - previous versions don't have that handling yet, so it depends how much we want to be backwards compatible for new libraries. For the migration it definitely needs to stay or else it's a breaking change for them (to their users)